Revert "Merge pull request #248 from Shopify/bugsnag-integration" (#289)
Shopify/maintenance_tasks: README.md

database. It also mounts Maintenance Tasks in your `config/routes.rb`. By
default the web UI can be accessed in the new `/maintenance_tasks` path.

> In case you use an exception reporting service (e.g. Bugsnag) you might want to
> define an error handler. See
> [Customizing the error handler](#customizing-the-error-handler) for more
> information.

Reviewer: ```suggestion
In case you use an exception reporting service (e.g. Bugsnag) you might want to
define an error handler. See [Customizing the error
handler](#customizing-the-error-handler) for more information.
```

Author: I'd rather not break the link in the end.

Exceptions raised while a Task is performing are rescued and information about
the error is persisted and visible in the UI.

> If you want to perform operations with errors raised during a Task execution,
> such as reporting them to an exception monitoring service like Bugsnag,
> you can define an error handler. Simply create a proc that takes an error as
> argument. Maintenance Tasks will then call the error handler proc whenever an
> error is raised.

Reviewer: IMO the original text from the revert was clearer, we may be able to improve on it but this is not.

```suggestion
If you want to integrate with an exception monitoring service (e.g. Bugsnag),
you can define an error handler:
```
